q2_types_genomics/reference_db/_format.py

```diff
@@ -294,3 +294,110 @@ class EggnogProteinSequencesDirFmt(model.DirectoryFormat):
 plugin.register_formats(EggnogProteinSequencesDirFmt)
 plugin.register_semantic_type_to_format(ReferenceDB[EggnogProteinSequences],
                                         EggnogProteinSequencesDirFmt)
+
+
+class BuscoGenericTextFileFmt(model.TextFileFormat):
+    def _validate_(self, level):
+        pass
+
+
+class BuscoGenericBinaryFileFmt(model.BinaryFileFormat):
+    def _validate_(self, level):
+        pass
+
+
+class BuscoDatabaseDirFmt(model.DirectoryFormat):
+    # File collections for text files
+    (
+        ancestral,
+        dataset,
+        lengths_cutoff,
+        scores_cutoff,
+        links_to_ODB10,
+        ancestral_variants,
+        ogs_id,
+        species,
+        prfls,
+        hmms,
+        refseq_db_md5
+    ) = [
+            model.FileCollection(pattern, format=BuscoGenericTextFileFmt)
+            for pattern in [
+                r'.+ancestral$',
+                r'.+dataset\.cfg$',
+                r'.+lengths_cutoff$',
+                r'.+scores_cutoff$',
+                r'.+links_to_ODB10\.txt$',
+                r'.+ancestral_variants$',
+                r'.+ogs\.id\.info$',
+                r'.+species\.info$',
+                r'.+\.prfl$',
+                r'.+\.hmm$',
+                r'.+refseq_db\.faa\.gz\.md5'
+            ]
+        ]
```

Comment:
Hey @misialq. I wanted to get your input on this first draft of the `BuscoDatabaseDirFmt`. I defined all file collections as `BuscoGenericTextFileFmt` but I was wondering if you'd like me to create more specific formats for some of these (e.g. `hmms` which are the HMMER files).